playbooks/deploy_registry.yml

```diff
@@ -1,11 +1,15 @@
 ---
 - name: Play to populate image_hashes for relevant images
   hosts: localhost
-  gather_facts: "{{ (setup_registry_service | default(True)) | bool }}"
+  gather_facts: False
   vars:
+    SETUP_REGISTRY_SERVICE: "{{ setup_registry_service | default(True)}}"
     destination_hosts:
       - registry_host
   pre_tasks:
```

Comment:
This is not a blocker but in this case `pre_tasks` would get replaced with `tasks` since there is no task section.